packages/modules/viewer-react/src/services/telemetry/Performance.ts

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { trackViewerMetric, viewerAI } from "./TelemetryService";
+
+export type PerformanceMeasures =
+  | "ViewerInitialized"
+  | "BaseViewerInitialized"
+  | "IModelConnected"
+  | "ViewStateCreated"
+  | "TileTreesLoaded";
```

Comment:
Going to be an interesting measure to track. Would probably want to check with paul or mark or the current perf tests guys how theyre handling this, if you havent already.

Reply:
this was the best method to track when tiles are loaded when I implemented, but I'll double check with the display team and update if necessary